components/style/values/computed/border.rs

```diff
@@ -81,3 +87,50 @@ impl ToAnimatedZero for BorderCornerRadius {
         Err(())
     }
 }
+
+/// https://drafts.csswg.org/css-backgrounds/#the-borderimage-repeat
+#[derive(Clone, Debug, MallocSizeOf, PartialEq)]
+pub struct BorderImageRepeat(RepeatKeyword, RepeatKeyword);
+
+impl BorderImageRepeat {
+    pub fn repeat() -> Self {
+        BorderImageRepeat(RepeatKeyword::Repeat, RepeatKeyword::Repeat)
+    }
+}
+
+impl ToCss for BorderImageRepeat {
+    fn to_css<W>(&self, dest: &mut W) -> fmt::Result
+    where
+        W: fmt::Write,
+    {
+        let BorderImageRepeat(horizontal, vertical) = *self;
```

Comment:
Now you need to actually serialize them, keeping the code that existed.

Reply:
Ok.Thank you for the review.I'll fix it